internal/args/unmarshal.go

```diff
@@ -101,31 +96,31 @@ func UnmarshalStruct(args []string, data interface{}) error {
 	return nil
 }
 
-func findFieldType(t reflect.Type, keys []string) (reflect.Type, error) {
+func fieldExist(t reflect.Type, argNameWords []string) bool {
 
 	switch {
-	case len(keys) == 0:
-		return t, nil
+	case len(argNameWords) == 0:
```

Comment:
In this case we should also check that we know how to unmarshal a type T.

This wil handle the case where argname should be `a.b.c.d=12` but we only type `a.b=12`

Reply:
This is already done at the beginning of the `set`